influxdb_client_3/__init__.py

```diff
@@ -147,7 +148,17 @@ def __init__(
 
         if query_port_overwrite is not None:
             port = query_port_overwrite
-        self._flight_client = FlightClient(f"grpc+tls://{hostname}:{port}", **self._flight_client_options)
+
+        gen_opts = [
+            ("grpc.secondary_user_agent", USER_AGENT)
+        ]
+
+        self._flight_client_options["generic_options"] = gen_opts
+
+        if scheme == 'https':
+            self._flight_client = FlightClient(f"grpc+tls://{hostname}:{port}", **self._flight_client_options)
+        else:
+            self._flight_client = FlightClient(f"grpc+tcp://{hostname}:{port}", **self._flight_client_options)
```

Comment:
Please simplify this to something like:
```suggestion
        if scheme == 'https':
            connectionString =  f"grpc+tls://{hostname}:{port}"
        else:
            connectionString = f"grpc+tcp://{hostname}:{port}"
        self._flight_client = FlightClient(connectionString, **self._flight_client_options)
```

Reply:
Yes.  The `connection_string` is a better solution.   Changed.